input/fsh/mappings/conceptMap-appointmentStatus-ajanvarauksenTila.fsh

```diff
@@ -0,0 +1,98 @@
+Instance: appointmentstatus-ajanvarauksentila
+InstanceOf: ConceptMap
+Title: "Appointment status to Ajanvarauksen tila"
+Description: "Mapping between FHIR Appoinment status codes and the Finnish logical model [*Ajanvaraus - Ajanvarauksen tila*](https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=1943) (oid `1.2.246.537.6.881`)."
+Usage: #definition
+* name = "FinnishAppointmentStatusCodeMap"
```

Comment:
Should we use "Fi" instead of "Finnish"?

Reply:
Hmmm... Good point.

Reply:
Fixed in https://github.com/fhir-fi/finnish-scheduling/pull/24/commits/2da00008e73a855473c9c06ba7cf6ca30b0bdd39